packages/zosjobs/src/SubmitJobs.ts

```diff
@@ -289,6 +300,172 @@ export class SubmitJobs {
             });
     }
 
+    /**
+     * Parse input string for JCL substitution
+     * @param {string} symbols - JCL substitution symbols
+     * @returns {IHeaderContent[]} headers - Headers to add to the request
+     * @memberof SubmitJobs
+     */
+    private static getSubstitutionHeaders(symbols: string): IHeaderContent[] {
+        const headers: IHeaderContent[] = [];
+        const blank = " ";
+        const equals = "=";
+        const maxSymLen = 8;
+        let symStartInx = 0;
+
+        moreSymLoop:
+        while(symStartInx < symbols.length) {
+            // skip all blanks at the start of a sym def
+            while (symbols[symStartInx] === blank) {
+                if (++symStartInx >= symbols.length) {
+                    break moreSymLoop;
```

Comment:
This is cool, I didn't know you could assign names to loops to break out of an outer loop 🚀

Reply:
Yeah, and SonarCloud hates it and wants me to remove the label. I am going to ignore SonarCloud.